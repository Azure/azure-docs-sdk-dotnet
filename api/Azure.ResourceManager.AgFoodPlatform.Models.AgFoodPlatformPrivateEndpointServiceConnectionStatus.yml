### YamlMime:NetType
type: struct
implementsWithMoniker:
- value: <xref href="System.IEquatable`1?alt=System.IEquatable&text=IEquatable" data-throw-if-not-resolved="True"/>&lt;<xref href="Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus?alt=Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus&text=AgFoodPlatformPrivateEndpointServiceConnectionStatus" data-throw-if-not-resolved="True"/>&gt;
inheritancesWithMoniker:
- values:
  - <xref href="System.ValueType?alt=System.ValueType&text=ValueType" data-throw-if-not-resolved="True"/>
constructors:
- uid: Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus.#ctor(System.String)
  monikers:
  - azure-dotnet-preview
operators:
- uid: Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus.op_Equality(Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus,Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus)
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus.op_Implicit(System.String)~Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus.op_Inequality(Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus,Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus)
  monikers:
  - azure-dotnet-preview
methods:
- uid: Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus.Equals(Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus)
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus.ToString
  monikers:
  - azure-dotnet-preview
properties:
- uid: Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus.Approved
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus.Pending
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus.Rejected
  monikers:
  - azure-dotnet-preview
uid: Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus
commentId: T:Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus
namespace: Azure.ResourceManager.AgFoodPlatform.Models
name: AgFoodPlatformPrivateEndpointServiceConnectionStatus
fullName: Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus
nameWithType: Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus
assembliesWithMoniker:
- value: Azure.ResourceManager.AgFoodPlatform.dll
packagesWithMoniker:
- value: Azure.ResourceManager.AgFoodPlatform v1.0.0-beta.3
syntaxWithMoniker:
- lang: csharp
  values:
  - value: 'public readonly struct AgFoodPlatformPrivateEndpointServiceConnectionStatus : IEquatable<Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus>'
- lang: fsharp
  values:
  - value: type AgFoodPlatformPrivateEndpointServiceConnectionStatus = struct
- lang: vb
  values:
  - value: >-
      Public Structure AgFoodPlatformPrivateEndpointServiceConnectionStatus

      Implements IEquatable(Of AgFoodPlatformPrivateEndpointServiceConnectionStatus)
devLangs:
- csharp
- vb
- fsharp
monikers:
- azure-dotnet-preview
summary: The private endpoint connection status.
metadata:
  api_name:
  - Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus
  api_location:
  - Azure.ResourceManager.AgFoodPlatform.dll
  topic_type:
  - apiref
  api_type:
  - Assembly
  f1_keywords:
  - Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus
  - Azure::ResourceManager::AgFoodPlatform::Models::AgFoodPlatformPrivateEndpointServiceConnectionStatus
  - AgFoodPlatformPrivateEndpointServiceConnectionStatus
  helpviewer_keywords:
  - Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus struct [.NET]
  monikers:
  - azure-dotnet-preview
