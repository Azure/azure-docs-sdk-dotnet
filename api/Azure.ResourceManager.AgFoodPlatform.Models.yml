### YamlMime:NetNamespace
classes:
- uid: Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateLinkServiceConnectionState
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AgFoodPlatform.Models.ArmAgFoodPlatformModelFactory
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AgFoodPlatform.Models.ArmAsyncOperation
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AgFoodPlatform.Models.CheckNameAvailabilityContent
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AgFoodPlatform.Models.CheckNameAvailabilityResponse
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AgFoodPlatform.Models.DetailedInformation
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AgFoodPlatform.Models.FarmBeatPatch
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AgFoodPlatform.Models.FarmBeatsUpdateProperties
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AgFoodPlatform.Models.SensorIntegration
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AgFoodPlatform.Models.UnitSystemsInfo
  monikers:
  - azure-dotnet-preview
structs:
- uid: Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointConnectionProvisioningState
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AgFoodPlatform.Models.CheckNameAvailabilityReason
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AgFoodPlatform.Models.ProvisioningState
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.AgFoodPlatform.Models.PublicNetworkAccess
  monikers:
  - azure-dotnet-preview
uid: Azure.ResourceManager.AgFoodPlatform.Models
commentId: N:Azure.ResourceManager.AgFoodPlatform.Models
name: Azure.ResourceManager.AgFoodPlatform.Models
fullName: Azure.ResourceManager.AgFoodPlatform.Models
devLangs:
- csharp
monikers:
- azure-dotnet-preview
metadata:
  f1_keywords:
  - Azure.ResourceManager.AgFoodPlatform.Models
  - Azure::ResourceManager::AgFoodPlatform::Models
  - Models
  - AgFoodPlatform.Models
  - AgFoodPlatform::Models
  helpviewer_keywords:
  - Azure.ResourceManager.AgFoodPlatform.Models namespace [.NET]
  monikers:
  - azure-dotnet-preview
