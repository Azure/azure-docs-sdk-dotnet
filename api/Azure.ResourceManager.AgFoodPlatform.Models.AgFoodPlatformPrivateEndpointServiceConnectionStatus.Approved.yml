### YamlMime:NetMember
type: property
members:
- returnsWithMoniker:
    type:
    - value: <xref href="Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus?alt=Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus&text=AgFoodPlatformPrivateEndpointServiceConnectionStatus" data-throw-if-not-resolved="True"/>
  type: property
  uid: Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus.Approved
  commentId: P:Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus.Approved
  name: Approved
  fullName: Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus.Approved
  nameWithType: AgFoodPlatformPrivateEndpointServiceConnectionStatus.Approved
  syntaxWithMoniker:
  - lang: csharp
    values:
    - value: public static Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus Approved { get; }
  - lang: fsharp
    values:
    - value: 'static member Approved : Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus'
  - lang: vb
    values:
    - value: Public Shared ReadOnly Property Approved As AgFoodPlatformPrivateEndpointServiceConnectionStatus
  monikers:
  - azure-dotnet-preview
  summary: Approved.
  metadata: {}
uid: Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus.Approved*
commentId: Overload:Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus.Approved
namespace: Azure.ResourceManager.AgFoodPlatform.Models
name: Approved
fullName: Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus.Approved
nameWithType: AgFoodPlatformPrivateEndpointServiceConnectionStatus.Approved
assembliesWithMoniker:
- value: Azure.ResourceManager.AgFoodPlatform.dll
packagesWithMoniker:
- value: Azure.ResourceManager.AgFoodPlatform v1.0.0-beta.3
devLangs:
- csharp
- vb
- fsharp
monikers:
- azure-dotnet-preview
metadata:
  api_name:
  - Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus.Approved
  - Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus.get_Approved
  api_location:
  - Azure.ResourceManager.AgFoodPlatform.dll
  topic_type:
  - apiref
  api_type:
  - Assembly
  f1_keywords:
  - Azure.ResourceManager.AgFoodPlatform.Models.AgFoodPlatformPrivateEndpointServiceConnectionStatus.Approved
  - Azure::ResourceManager::AgFoodPlatform::Models::AgFoodPlatformPrivateEndpointServiceConnectionStatus::Approved
  - Approved
  - AgFoodPlatformPrivateEndpointServiceConnectionStatus.Approved
  - AgFoodPlatformPrivateEndpointServiceConnectionStatus::Approved
  helpviewer_keywords:
  - AgFoodPlatformPrivateEndpointServiceConnectionStatus.Approved property [.NET]
  - Approved property [.NET], struct AgFoodPlatformPrivateEndpointServiceConnectionStatus
  monikers:
  - azure-dotnet-preview
